src/components/page-layout/layout.config.js

```diff
@@ -5,20 +5,17 @@ export default {
   },
   accordion: {
     sidebar: 'accordion', // NavigationAccordion sidebar
```

Comment:
I think we can remove the `sidebar` configuration option. In my opinion, there should only be one sidebar with no customizations (since this is the site's main navigation and we didn't have offer customization with NavTabs). We'll move customization options to the PageLayout aside.

Reply:
We can save making these updates for a later PR.